tests/unittests/tests-lora_serialization/Makefile.include

```diff
@@ -0,0 +1 @@
+USEPKG += lora-serialization
```

Comment:
I know there are already other `pkg`s in this test application, but `embunit` can be used outside of the `unittests`, so can you maybe move them to a separate application?

Reply:
Moved it to a separate application.